dune

```diff
@@ -1,19 +1,108 @@
 (library
  (name stdcompat)
  (public_name stdcompat)
- (modules stdcompat__native stdcompat__init stdcompat__root stdcompat__seq
+ (flags (:standard -nolabels))
```

Comment:
Why do you need `-nolabels` here?

Reply:
Because the code uses the same interface files for `ArrayLabels`, `StringLabels` as the normal `Array`, `String` (etc), so to preserve how the original code does the compilation I need `-nolabels`.